recipes/libtheora/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "1.1.1" %}
+
+package:
+  name: libtheora
+  version: {{ version }}
+
+source:
+  fn: libtheora-{{ version }}.tar.bz2
+  url: http://downloads.xiph.org/releases/theora/libtheora-{{ version }}.tar.bz2
+  md5: 292ab65cedd5021d6b7ddd117e07cd8e
```

Comment:
It looks like they have sha256 checksums. Could we use one of those instead? md5 has been proven to be a weak checksum.

ref: http://downloads.xiph.org/releases/theora/SHA256SUMS